Client-Side/shecodesprogramming/src/Components/HomePage/homepage.css

```diff
@@ -0,0 +1,126 @@
+.layout{
+    width: 95%;
+}
+
+.MainLayout{
+    display: flex;
+}
+
+.welcome{
+    width: 55%;
+    margin-left: 25%;
+    margin-top: 70px;
+}
+
+p{
+    margin-left: 5%;
+    line-height: 25px;
+}
+
+.sublayout{
+    width: 80%;
+    margin-left: 10%;
+    text-align: left;
+}
+
+.display{
+    display: flex;
+}
+
+.text-image{
+    margin-top: 250px;
```

Comment:
Please use selectors and combinators

Reply:
I added selectors and combinators could you please review it.